contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcViewConstants.java

```diff
@@ -99,7 +98,7 @@
               4096.0, 8192.0, 16384.0, 32768.0, 65536.0));
 
   // Use Aggregation.Mean to record sum and count stats at the same time.
-  @VisibleForTesting static final Aggregation MEAN = Mean.create();
+  @VisibleForTesting static final Aggregation MEAN = Aggregation.Mean.create();
```

Comment:
How will we handle the existing Mean RPC views?

Reply:
Mean is only used in interval views, and those views are not exported to any backends. I think we can remove those views along with Mean and Window in the future.

Reply:
The error count views also use Mean, though I think they also need to be updated to use different Measures, so we can update the Aggregation type at the same time.

Reply:
Oh correct, the error count views should probably be updated to use Count instead.